src/marshal.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(not(Py_LIMITED_API))]
```

Comment:
Could be worth adding a module docstring here and stating that this is not supported with the limited api.

Reply:
Good idea.